src/main/java/core/basesyntax/service/CustomeException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class CustomeException extends RuntimeException {
```

Comment:
There is a typo in the class name `CustomeException`. It should be `CustomException`.